### DISCUSSION TITLE
Add property for fiscal year

### DISCUSSION THREAD

**Discussion by ceboxsell** [teslamate-org/teslamate#4008] [Ideas]
It would be good to have a property to set the start month of the fiscal year. Currently the default seems to be month 0.
For Australia this is month 7.

**Comment by cwanja** [comment]
Can you expand on this? Where you need this added?

**Reply by ceboxsell** [reply]
Sorry, I thought it was obvious. 
Just like selecting Celcius/F or Km/miles we have a setting for the start month of the financial year. 

When I searched for fiscal I found that the reports had the value 0 hard coded. 

My thought was if this was possible then that value would be set by the property so fiscal year starts at 7 from the property.

The current dashboards JSON show this,

"fiscalYearStartMonth": 0

**Reply by cwanja** [reply]
What dashboard uses fiscal year in the dashboard or calculation?

**Reply by ceboxsell** [reply]
As this is a date period selection it may be grafana related. It applies to any dashboard really.  I will look into Grafana controlling this so don't worry about it.

<img width="470" alt="image" src="https://github.com/teslamate-org/teslamate/assets/10136084/bd8b561d-895f-4991-b53f-65f4d9e8df06">
